sections/performance/nativeoverutil.md

```diff
@@ -0,0 +1,76 @@
+# Prefer native JS methods over user-land utils like Lodash
+
+
+<br/><br/>
+
+### One Paragraph Explainer
+
+Sometimes, using native methods is better than requiring `lodash` or `underscore` because it **will not lead in a performance boost** and use more space than necessary.
+The performance using native methods result in an overall ~50% gain which includes the following methods:
+ - `Array.concat`
+ - `Array.fill`
+ - `Array.filter`
+ - `Array.map`
+ - `(Array|String).indexOf`
+ - `Object.find`
+ - ...
+
+
+<!-- comp here: https://gist.github.com/Berkmann18/3a99f308d58535ab0719ac8fc3c3b8bb-->
+
+<br/><br/>
+
+### Example: benchmark comparison - Lodash vs V8 (Native)
+The graph below shows the mean of the benchmarks for a variety of Lodash methods, this shows that Lodash methods take on average 146.23% more time to complete the same tasks as V8 methods.
+
+![meanDiag](../../assets/images/sampleMeanDiag.png)
+
+### Code Example – Benchmark test on `_.concat`/`Array.concat`
+```javascript
+const _ = require('lodash'),
+  __ = require('underscore'),
+  Suite = require('benchmark').Suite,
+  opts = require('./utils'); //cf. https://github.com/Berkmann18/NativeVsUtils/blob/master/utils.js
+
+const concatSuite = new Suite('concat', opts);
+const array = [0, 1, 2];
+
+concatSuite.add('lodash', () => _.concat(array, 3, 4, 5))
+  .add('underscore', () => __.concat(array, 3, 4, 5))
+  .add('native', () => array.concat(3, 4, 5))
+  .run({ 'async': true });
+```
+
+Which returns this:
+
+![output](../../assets/images/concat-benchmark.png)
+
+You can find a bigger list of benchmarks [here](https://github.com/Berkmann18/NativeVsUtils/blob/master/index.txt) or alternatively [run this](https://github.com/Berkmann18/NativeVsUtils/blob/master/index.js) which would show the same but with colours.
+
+### "You don't (may not) need Lodash/Underscore"
```

Comment:
Add "Blog Quote:" before the title, [see other bullets for example](https://github.com/i0natan/nodebestpractices/blob/master/sections/projectstructre/breakintcomponents.md)